ncj/maya/render-arnold-windows/job.template.json

```diff
@@ -17,6 +17,17 @@
                 "description": "The unique name of the job."
             }
         },
+        "mayaRenderer": {
```

Comment:
mayaVersion?

Reply:
yep

---
In reply to: [191115925](https://github.com/Azure/BatchLabs-data/pull/16#discussion_r191115925) [](ancestors = 191115925)